src/components/Journal/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const Journal = ({ question, questionTitle }) => {
+  return (
+    <div className="journal">
+      <section className="journal__header">
+        <h5 className="journal__question-title">
+          {`${questionTitle} `}
```

Comment:
Why template literals? 
```suggestion
          {questionTitle}
```

Reply:
It is easier to me than adding margin =D